CHANGELOG.md

```diff
@@ -13,7 +13,8 @@ CHANGELOG
 - Add support for customizing the cluster Slurm configuration via the ParallelCluster configuration YAML file.
 - Track the longest dynamic node idle time in CloudWatch Dashboard.
 - Add new configuration section `HealthChecks/Gpu` for enabling the GPU Health Check in the compute node before job execution.
-- Add support for `Tags` in the `SlurmQueues` section.
+- Add new configuration section `HealthChecks/Gpu` for enabling the GPU Health Check in the compute node before job submission.
+- Add support for `Tags` in the `SlurmQueues` and `SlurmQueues/ComputeResources` section.
```

Comment:
Is the line "Add new configuration section `HealthChecks/Gpu` for enabling ..." duplicated here?